### PR TITLE
docs(content): add missing links + remove lorem ipsum

### PR DESCRIPTION
**Pull request by bobylito**

### CODE REVIEW

File: docs/documentation.md [hunk 1]
```diff
@@ -1188,17 +1188,19 @@ If your code base is relying on an other framework, feel free to use it to creat
...
 
+#### Example with vanillaJS
 
+To extend the capabilities of instantsearch.js, you don't need any to use any dependencies. For example, you could use only plain javascript. Have a look at our simple js example [<i class='fa fa-github'></i> instantsearch/instantsearch-JavaScript-widget](https://github.com/instantsearch/instantsearch-JavaScript-widget) to get started.
 
 #### Example with jQuery
 {:.no-toc}
 
-You may want to use an existing jQuery library to power your custom widget: that's definitely doable. Take a look at this jQuery-based custom widgets we've done: [<i class="fa fa-github"></i> algolia/instantsearch-ion.rangeSlider](https://github.com/algolia/instantsearch-ion.rangeSlider).
+You may want to use an existing jQuery library to power your custom widget: that's definitely doable. Take a look at this jQuery-based custom widgets we've done: [<i class="fa fa-github"></i> instantsearch/instantsearch-ion.rangeSlider](https://github.com/instantsearch/instantsearch-ion.rangeSlider).
```
Comment (vvo) [inline docs/documentation.md]: Can we do the same than React? => simple example, have a look at a more complete..

File: docs/documentation.md [hunk 2]
```diff
@@ -1188,17 +1188,19 @@ If your code base is relying on an other framework, feel free to use it to creat
 
 </div>
 
+#### Example with vanillaJS
 
+To extend the capabilities of instantsearch.js, you don't need any to use any dependencies. For example, you could use only plain javascript. Have a look at our simple js example [<i class='fa fa-github'></i> instantsearch/instantsearch-JavaScript-widget](https://github.com/instantsearch/instantsearch-JavaScript-widget) to get started.
```
Comment (vvo) [inline docs/documentation.md]: typo "you don't need any to use any"

File: docs/examples.haml [hunk 3]
```diff
@@ -17,7 +17,7 @@ permalink: /examples/
         %img.m-b.img-thumbnail.pull-left.m-r{src: "{{ "/examples/media/capture.png" | prepend: site.baseurl }}", width: 270}
       %a{href: '{{ "/examples/media/" | prepend: site.baseurl }}'}
         %h3 Media
-      %p Lorem ipsum dolor sit amet, consectetur adipisicing elit. Quo blanditiis ad, quidem.
+      %p Find the perfect movie, with this video platform example.
```
Comment (vvo) [inline docs/examples.haml]: I think @lae25c has some more edit, no big deal

File: docs/documentation.md (hunk 1 above)
Comment (pixelastic) [inline docs/documentation.md]: Typo: "Take a look at this jQuery-based custom widget__s__"

File: docs/documentation.md [hunk 4]
```diff
@@ -1188,17 +1188,19 @@ If your code base is relying on an other framework, feel free to use it to creat
...
+You may want to use an existing jQuery library to power your custom widget: that's definitely doable. Take a look at this jQuery-based custom widgets we've done: [<i class="fa fa-github"></i> instantsearch/instantsearch-ion.rangeSlider](https://github.com/instantsearch/instantsearch-ion.rangeSlider).
 
 #### Example with React
 {:.no-toc}
 
-Lorem ipsum dolor sit amet, consectetur adipisicing elit. Deleniti perspiciatis suscipit voluptas esse quam fugiat recusandae harum, illum, sint animi excepturi at. Aliquid accusamus ratione atque sit ipsam quia itaque.
+You can also use React for your next widget. Have a look at a simple example, [<i class='fa fa-github'></i> instantsearch/instantsearch-React-widget ](https://github.com/instantsearch/instantsearch-React-widget), or dig into a full fledge React component integration, [<i class='fa fa-github'></i> instantsearch/instantsearch-googlemaps](https://github.com/instantsearch/instantsearch-googlemaps).
```
Comment (pixelastic) [inline docs/documentation.md]: Isn't it "full fledged"?

Comment (bobylito) [inline docs/documentation.md]: Sure :)

**Pull request merged**